scripts/hf_upload.py

```diff
@@ -0,0 +1,18 @@
+import os
+from datetime import datetime
+from huggingface_hub import login, HfFileSystem
+import forecast_csv
+
+if __name__ == "__main__":
+
+    hf_token = os.getenv("HF_TOKEN")
+    hf_repo = os.getenv("HF_REPO")
+
+    now = datetime.utcnow()
+    forecast = forecast_csv.forecast_for_site(latitude=51.75, longitude=-1.25, capacity_kwp=1.25, init_time=now)
```

Comment:
could we also loop over the different models, and add the model name to the filename we save it at?